CS/DockPanelInCode_Ex/Window1.xaml

```diff
@@ -1,31 +1,48 @@
-<Window x:Class="DockPanelInCode_Ex.Window1" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" Title="Window1" Height="300" Width="640" xmlns:dxdo="http://schemas.devexpress.com/winfx/2008/xaml/docking" xmlns:dxcore="http://schemas.devexpress.com/winfx/2008/xaml/core">
+<Window 
+    x:Class="DockPanelInCode_Ex.Window1"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:dxcore="http://schemas.devexpress.com/winfx/2008/xaml/core"
+    xmlns:dxdo="http://schemas.devexpress.com/winfx/2008/xaml/docking"
+    Title="Window1"
+    Width="640"
+    Height="300">
     <Grid>
-        <dxdo:DockLayoutManager Margin="12" Name="dockManager1" dxcore:ThemeManager.ThemeName="Office2007Blue">
+        <dxdo:DockLayoutManager
+            Name="dockManager1"
+            Margin="12"
+            dxcore:ThemeManager.ThemeName="Office2019Colorful">
             <dxdo:LayoutGroup x:Name="rootGroup" Orientation="Horizontal">
-                <dxdo:LayoutPanel x:Name="paneOutput" Caption="Output" ItemWidth="250">
+                <dxdo:LayoutPanel
+                    x:Name="paneOutput"
+                    Caption="Output"
+                    ItemWidth="250">
                     <RichTextBox />
                 </dxdo:LayoutPanel>
-                
                 <dxdo:DocumentGroup x:Name="documentGroup1">
                     <dxdo:DocumentPanel x:Name="paneDocument1" Caption="Document 1">
                         <RichTextBox />
                     </dxdo:DocumentPanel>
                 </dxdo:DocumentGroup>
-                
             </dxdo:LayoutGroup>
-            
             <dxdo:DockLayoutManager.FloatGroups>
-                <dxdo:FloatGroup FloatSize="200,150" FloatLocation="100,50">
+                <dxdo:FloatGroup FloatLocation="100,50" FloatSize="200,150">
                     <dxdo:LayoutPanel x:Name="paneProperties" Caption="Properties">
                         <DockPanel>
-                            <Button Content="Dock making a split container" DockPanel.Dock="Top" Height="40" Click="Button1_Click" />
-                            <Button Content="Dock making a tab container" DockPanel.Dock="Top" Height="40" Click="Button2_Click" />
+                            <Button
+                                Height="40"
+                                Click="Button1_Click"
+                                Content="Dock making a split container"
```

Comment:
Is this string editable? If so, what do you mean by "making" in this context?

Reply:
```suggestion
                                Content="Dock a panel to a split container"
```